apps/web/modules/ee/audit-logs/lib/logger.ts

```diff
@@ -0,0 +1,25 @@
+import { AUDIT_LOG_ENABLED, AUDIT_LOG_PATH } from "@/lib/constants";
+import Pino, { type Logger, type LoggerOptions } from "pino";
+
+// Ensure the audit logger has a consistent format
+const auditLoggerConfig: LoggerOptions = {
+  level: "info",
+  base: undefined,
+  timestamp: false,
+  formatters: {
+    level: (label) => ({ level: label }),
+  },
+  transport: {
+    target: "pino/file",
+    options: {
+      destination: AUDIT_LOG_PATH,
+      mkdir: true,
+      append: true,
```

Comment:
Is it possible to also set the file permissions here, e.g. using restrictive permissions (600 or 640) on audit log files?
If so, please add :-)